composer.json

```diff
@@ -16,26 +16,28 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": "^5.6|^7.0",
         "composer/installers": "^1.2",
         "cweagans/composer-patches": "^1.6.5",
         "drupal-composer/drupal-scaffold": "^2.5",
         "drupal/console": "^1.0.2",
-        "drupal/core": "^8.7.0",
-        "drush/drush": "^9.0.0",
+        "drupal/core": "^8.7",
+        "drush/drush": "^9",
         "vlucas/phpdotenv": "^2.4",
-        "webflo/drupal-finder": "^1.0.0",
+        "webflo/drupal-finder": "^1",
         "webmozart/path-util": "^2.3",
-        "zaporylie/composer-drupal-optimizations": "^1.0"
+        "zaporylie/composer-drupal-optimizations": "^1"
     },
     "require-dev": {
-        "webflo/drupal-core-require-dev": "^8.7.0"
+        "behat/mink": "^1.7@dev",
+        "behat/mink-selenium2-driver": "^1.3@dev",
+        "friendsofphp/php-cs-fixer": "^2.15",
+        "phpunit/phpunit": "^6.5",
+        "webflo/drupal-core-require-dev": "^8.7"
```

Comment:
The whitelist is required in order to avoid unintended burden of the stability with dangerous dev dependencies, so a more narrow decision is to allow explicitly the non stable packages using [stability flags](https://getcomposer.org/doc/04-schema.md#package-links).